benchmark/cleanup-cluster

```diff
@@ -4,11 +4,11 @@
 Clean up all containers and datasets in a cluster.
 """
 
-from _preamble import TOPLEVEL, BASEPATH
+from admin._preamble import TOPLEVEL, BASEPATH
```

Comment:
These values are not used in `cluster_cleanup.main`. Remove these two parameters from `main`.